frontend/lib/about/language_change.dart

```diff
@@ -28,17 +26,12 @@ class LanguageChange extends StatelessWidget {
     ]);
   }
 
-  switchLanguage(BuildContext context, String language) {
+  Future<void> switchLanguage(BuildContext context, String language) async {
     final messengerState = ScaffoldMessenger.of(context);
     final settings = Provider.of<SettingsModel>(context, listen: false);
     LocaleSettings.setLocaleRaw(language);
-    settings.setLanguage(language: language);
-    Navigator.push(
-      context,
-      AppRoute(
-        builder: (context) => HomePage(),
-      ),
-    );
+    await settings.setLanguage(language: language);
+    Navigator.pop(context);
```

Comment:
this change will avoid showing the snackbar of the language change.
You could move the showSnackbar before the `settings.setLanguage`
Not sure if there is a better solution

Reply:
Let's merge #1305 first and check again. I suspect, that the tree recreation removes the snackbar.

Reply:
updated the branch. just some conflicts to resolve

Reply:
Snackbars are visible for me